frontend/ui/data/optionsutil.lua

```diff
@@ -101,7 +101,10 @@ function optionsutil.showValues(configurable, option, prefix, document)
     end
     if option.help_text_func then
         -- Allow for concatenating a dynamic help_text_func to a static help_text
-        help_text = T("%1\n%2\n", help_text, option.help_text_func(configurable, document))
+        local more_text = option.help_text_func(configurable, document)
+        if more_text and more_text ~= "" then
+            help_text = T("%1\n%2\n", help_text, more_text)
```

Comment:
```suggestion
            help_text = T("%1\n%2", help_text, more_text) .. "\n"
```

Reply:
That thing is not translated (no `_()`), so it shouldn't really matter either way, right ?

Reply:
Oops, my bad. I guess I was expecting string.format for that. :)